src/math/calculation.js

```diff
@@ -829,4 +829,58 @@ function hypot(x, y, z) {
   return Math.sqrt(sum) * max;
 }
 
+/**
+ * Calculates the fractional part of a number.
+ *
+ * @method fract
+ * @param {Number} num Number whose fractional part needs to be found out
+ * @returns {Number} fractional part of x, i.e, {x}
+ * @example
+ * <div>
+ * <code>
+ * function setup() {
+ *   createCanvas(windowWidth, windowHeight);
+ *   fill(0);
+ *   let nums = [
+ *     Infinity,
+ *     1e-10,
+ *     1.4215e-10,
+ *     14213.214e-10,
+ *     142144.42142241241212521e10,
+ *     133232e10,
+ *     1.2412,
+ *     '241.12421'
+ *   ];
+ *   let spacing = 150;
+ *   for (let i = 0; i < nums.length; i++) {
+ *     text(String(nums[i]), i * spacing, 50);
+ *     text(String(fract(nums[i])), i * spacing, 100);
+ *   }
```

Comment:
this example might be a bit hard for beginners to understand because it makes use of arrays and for loops. can you update to make this an example that does a single fract() command?